# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/thunderbird/hotfix-stdint.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

comm/third_party/rnp/src/libsexp/include/sexp/sexp-error.h:66:5: error: 'uint32_t' does not name a type
 6:19.02    66 |     uint32_t            get_position(void) const { return position; };
 6:19.02       |     ^~~~~~~~

--- thunderbird-115.0/comm/third_party/rnp/src/libsexp/include/sexp/sexp-error.h.vanilla	2023-07-20 20:33:59.888953270 +0200
+++ thunderbird-115.0/comm/third_party/rnp/src/libsexp/include/sexp/sexp-error.h	2023-07-20 20:34:14.187952440 +0200
@@ -29,6 +29,7 @@
 
 #pragma once
 
+#include <stdint.h>
 #include <exception>
 #include <iostream>
 #include <string>
